packages/venia-concept/src/components/Checkout/exit.js

```diff
@@ -0,0 +1,32 @@
+import { Component, createElement } from 'react';
+import { func, shape, string } from 'prop-types';
+
+import classify from 'src/classify';
+import ResetButton from './resetButton';
+import defaultClasses from './exit.css';
+
+class Exit extends Component {
+    static propTypes = {
+        classes: shape({
+            body: string,
+            footer: string,
+            root: string
+        }),
+        resetCheckout: func
```

Comment:
Should this prop be required?

Reply:
Yes, probably. I'll do a sweep and see if I can make some props required. 👍